server/bootstrap/src/org/labkey/bootstrap/LabKeyBootstrapClassLoader.java

```diff
@@ -56,10 +55,6 @@ public class LabKeyBootstrapClassLoader extends WebappClassLoader implements Exp
         String headless = "java.awt.headless";
         if (System.getProperty(headless) == null)
             System.setProperty(headless, "true");
-        // On most installs, catalina.home and catalina.base point to the same directory. However, it's possible
-        // to have multiple instances share the Tomcat binaries but have their own ./logs, ./conf, etc directories
-        // Thus, we want to use catalina.base for our place to find log files. http://www.jguru.com/faq/view.jsp?EID=1121565
-        PipelineBootstrapConfig.ensureLogHomeSet(System.getProperty("catalina.base") + "/logs");
```

Comment:
You've moved this for the embedded case, but I don't think it's being run for standalone Tomcat. I'm not aware of anyone currently deploying Tomcat and the webapp in the way described here, but seems good to retain until we're 100% embedded.

Reply:
See other PR where this is addressed